lib/renderer/api/exports/module-list.js

```diff
@@ -0,0 +1,7 @@
+module.exports = [
```

Comment:
Does putting this file in this folder allow apps to require it via `require('module-list')`? I thought this folder was on the global module path.